nbs/core.ipynb

```diff
@@ -64,9 +64,17 @@
     "def _build_fn_name(fn) -> str:\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #48.                   MinTrace(method='mint_shrink')]
</pre><p>Did you change the example from <code>ols</code> to <code>mint_shrink</code> ?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/hierarchicalforecast/pull/102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>I changed it to make sure that <code>MinTrace</code> was giving the intended model name in the <code>Y_rec_df</code>.</p>